### ISSUE TITLE
Custom Confirmation Modal

### ISSUE THREAD

**Issue by ozergul** [ngneat/dialog#7]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[ ] Feature request
[ ] Documentation issue or request
[x] Support request
[ ] Other... Please describe:
</code></pre>

## Current behavior

Cannot pass data to global modal.

## Expected behavior

Be able to pass data to global modal.

## Environment

<pre><code>
Angular version: 10.0.1

Browser:
- [ x] Chrome
 
For Tooling issues:
- Node version: v12.18.0
- Platform:  Windows 

</code></pre>

My module:
```
    DialogModule.forRoot({
      confirm: {
        component: ConfirmationModalComponent,
      },
    }),
```

My confirmation modal:
```
import { DialogRef } from '@ngneat/dialog';
import { ChangeDetectionStrategy, Component } from '@angular/core';
import { DialogContentSymbol } from '@ngneat/dialog/lib/types'; // <--------------------- 

@Component({
  template: `
    <div class="modal-content">
      <div class="modal-header">
        <ng-container [ngSwitch]="title?.type">
          <h5 *ngSwitchCase="'string'" class="modal-title" [innerHTML]="title.content | translate"></h5>
          <h5 *ngSwitchCase="'template'" class="modal-title">
            <ng-container *ngTemplateOutlet="title.content; context: context"></ng-container>
          </h5>
        </ng-container>
      </div>
      <div class="modal-body" [ngSwitch]="body.type">
        <p *ngSwitchCase="'string'" [class.with-title]="title" [innerHTML]="body.content"></p>
        <p *ngSwitchCase="'template'" [class.with-title]="title">
          <ng-container *ngTemplateOutlet="body.content; context: context"></ng-container>
        </p>
      </div>
      <div class="modal-footer">
        <button type="button" class="btn btn-primary" (click)="ref.close(true)">Confirm</button>
        <button type="button" class="btn btn-secondary" data-dismiss="modal" (click)="ref.close(false)">Close</button>
      </div>
    </div>
  `,
  changeDetection: ChangeDetectionStrategy.OnPush,
})
export class ConfirmationModalComponent {
  title = this.ref.data[DialogContentSymbol].title;
  body = this.ref.data[DialogContentSymbol].body;
  context = {
    $implicit: this.ref,
    data: this.ref.data,
  };

  constructor(public ref: DialogRef) {}
}
```

Actually I wanted to create custom modal component with bootstrap. As I understand `DialogContentSymbol` not exported. What's the solution to use custom component.

**Comment by NetanelBasal** [comment]
@tonivj5

**Comment by tonivj5** [comment]
Good catch! Yeah, it should be exposed somehow

I'll do a PR :+1:

**Comment by tonivj5** [comment]
@ozergul it should be fixed by #8 and I have added your custom confirm dialog to the demo page :+1:

**Comment by ozergul** [comment]
Thanks for fast fix 👍
